content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,71 @@
+| Servers      | All          | Environment (Development, QA, Production) <br> Server Tags <br> Servers (List of all servers)                         |
+| Rules        | All          | Severity (Critical, High, Medium, Low, Note) <br> Vulnerability Tags <br> Vulnerability Rules (List of all rules)                         |
+
+
+<!-- Add info re: build number option - App > Vuln trend -->
+
+## Save Reports
+
+You can save filter criteria to recall any customized report at a later time. Saved reports are at the User level, so each of you have your own defined list of saved vulnerability trend reports. You can edit or delete these reports at any time. 
+
+<a href="assets/images/vulnerability-trend-report-name.png" rel="lightbox" title="Save a report"><img class="thumbnail" src="assets/images/vulnerability-trend-report-name.png"/></a> 
+ 
+To save a report view, click the star icon at the top right of the graphs. This generates a popup with a field to name the report. Once saved, the named report appears next to the **Vulnerability Trend** heading with a dropdown menu. Each time you come to the Vulnerability Trend page, the menu shows all of your saved reports as well as an option to **Start a new report**. 
+
+<a href="assets/images/Vulnerability-trend-save-report.png" rel="lightbox" title="Menu of saved reports"><img class="thumbnail" src="assets/images/Vulnerability-trend-save-report.png"/></a> 
+
+### Rename reports
+
+When viewing a saved report, hover over the star icon to generate a **Manage Report** tooltip. Click the icon to produce a popup with a field to rename the report and buttons to **Cancel**, **Remove** or **Save**. 
+
+### Edit and remove reports
+
+If you change filter options while viewing a saved report, the star icon appears and the report name is updated as **Edited**. Click the icon to generate a popup menu to **Save Existing** or **Save As New**. Choose **Save Existing** to update the saved report name with the current filters and remove the **Edited** status. Choose **Save As New** to save the report view with the current filters as a new report under a different name.
+
+Click **Remove** to permanently delete the saved report that you're currently viewing. Contrast automatically takes you to the default Vulnerability Trend page view and removes the report name from the dropdown menu. 
+
+### Start new reports
+
+To clear unsaved edits to an existing report and start over with the report defaults, choose the **Start a new report** option in the dropdown menu. The report name changes to **New Report**.
+
+### Manage reports 
+
+In the dropdown menu, click the **Manage** link to rename or remove saved reports. Select the checkbox by each report that you want to rename or remove in the **Manage Saved Reports** dialog. You can also select a checkbox to select all saved reports and edit as a batch action. 
```

Comment:
This isn't quite accurate. Let's try something more like: "When more than five saved reports have been created, a Manage link will appear within the Saved Reports dropdown. This will bring you to the Manage Saved Reports dialog. Select the checkbox next to each report that you want to remove. To rename, click the report name and edit inline. You will also have the option to search for reports or utilize a "Select All" checkbox for negative selection." (Checking the box is only for deleting - you can't edit as a batch action, and there have to be more than 5 reports before this functionality appears.)